web_client/panels/ZoomWidget.js

```diff
@@ -166,6 +180,74 @@ var ZoomWidget = Panel.extend({
         this._zoomSliderChange();
     },
 
+     /**
+     * A handler called when the download view button is clicked.
+     */
+    _downloadView(evt) {
+        var imageId = router.getQuery('image');
+        var bounds = router.getQuery('bounds');
+        var boundsTab = bounds.split(',');
```

Comment:
There may not be any bounds in the route if an image was just loaded.

Reply:
I'm not sure to see what you're talking about, I tried to reproduce this in downloading an image that was just upload, this didn't cause any issue, and the bounds were set. What kind of changes could be more appreciate here ?

Reply:
I only saw it when I tried to merge the branch into master and get the tests to pass.  There was a merge conflict with some of the tests, and I may have not resolved them appropriately.

Reply:
We probably should be using `this.viewer.bounds()` rather then reparsing the router information.

Reply:
I'm resolving the conflict and fix the `annotations test`, I will see how I can use the `viewer.bounds` rather than using the router information. Thank you !